### PR TITLE
Skill_Tutorial.md updated

### LINKED ISSUE #2944: Update links in susi.ai skill devolopment tutorial
**Actual Behaviour**

1. The links for CMS is not directing to the correct create skill page.

**Expected Behaviour**

1. It should direct to Create skill page. 

2. Also some documentation changes must be made for previewing the changes.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Dilshaad21**
Fixes #2944 

Changes: 

**Following changes made**
1. Susi.ai CMS link is updated to direct to the create skill page.
2. DIrections to open susi.ai public chat to view final changes.

### CODE REVIEW

File: docs/Skill_Tutorial.md [hunk 1]
```diff
@@ -11,17 +11,18 @@ It's easy, DON'T PANIC. You don't need to be a software developer to enhance SUS
...
+- your skill will be used by SUSI for all requests of all users! (unless you create a private skill)
 - other people can see and edit what you just wrote into the skill CMS
 
 ### What you have to do
 
-We have a SUSI Skill CMS at https://skills.susi.ai. Here you can create public SUSI.AI skills in a simpled editor window or private skill bots. Both can be created in the same way except that one is public and can be edited by anyone and the other is private, which can be edited only by you. The private skill bot also provides a Javascript embeded code, which you can add to your website. You can configure colors and icons to suite your style.
+We have a SUSI Skill CMS at https://susi.ai/skillWizard. Here you can create public SUSI.AI skills in a simpled editor window or private skill bots and check the new changes immediately in the preview at the right. Both can be created in the same way except that one is public and can be edited by anyone and the other is private, which can be edited only by you. The private skill bot also provides a Javascript embeded code, which you can add to your website. You can configure colors and icons to suite your style.
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Make it `We have a SUSI Skill Wizard`

File: docs/Skill_Tutorial.md [hunk 2]
```diff
@@ -11,17 +11,18 @@ It's easy, DON'T PANIC. You don't need to be a software developer to enhance SUS
...
 
 ### Preparation to start testing
 
-To test the SUSI.AI Skills you are editing, you can use the public chat at:
-https://chat.susi.ai
+To test the SUSI.AI Skills once you have created, you can use the public chat at:
```
Comment (akshatnitd) [inline docs/Skill_Tutorial.md]: Make it
```
at: https://susi.ai
Click at the chat
```

File: docs/Skill_Tutorial.md (hunk 1 above)
Comment (Dilshaad21) [reply docs/Skill_Tutorial.md]: Should I make susi skill wizard wherever it's written susi skill cms

**Comment by Dilshaad21** [comment]
@akshatnitd also the same changes must be made to https://github.com/fossasia/susi_skill_cms/blob/master/docs/Skill_Tutorial.md.

**Comment by mariobehling** [comment]
> @akshatnitd also the same changes must be made to https://github.com/fossasia/susi_skill_cms/blob/master/docs/Skill_Tutorial.md.

We only use one place for the tutorial now, which is https://github.com/fossasia/susi.ai/blob/master/docs/Skill_Tutorial.md

Thanks

**Pull request merged**